src/constants.ts

```diff
@@ -59,9 +59,7 @@ const JOB_TYPE = {
   COMPARE_USER_INFO: 9,
 };
 
-const sidServerMapping = {
+export const sidServerMapping : { [k:string | number]: string } = {
```

Comment:
Is there a way we can make this type strictly `number` or strictly `string`?

Reply:
sure, will have to make the changes to see